app/assets/javascripts/models/project.js

```diff
@@ -18,9 +19,8 @@ module.exports = Backbone.Model.extend({
 
     this.on('change:last_changeset_id', this.updateChangesets, this);
 
-    this.stories = new StoryCollection();
-    this.stories.url = this.url() + '/stories';
-    this.stories.project = this;
+    this.projectBoard = new ProjectBoard({ project: this });
+    this.projectBoard.stories.url = this.url() + '/stories';
```

Comment:
I think string interpolation could be better here. Like that: `${this.url()}/stories`, just following the pattern that you guys have followed in the project and becoming more secure.